app/services/appropriate_bodies/participants_filter.rb

```diff
@@ -49,23 +43,6 @@ def filter_query(scoped, query)
       ).ransack("#{fields}_cont": query).result.distinct
```

Comment:
do we need to cut down some of those fields or should we keep them all?

Reply:
Sure, will look to reduce the fields